### PR TITLE
Save feature reports method

### PR DESCRIPTION
**Pull request by rawadelkontar**
### Pull Request Description

[description here]

### Checklist (Delete lines that don't apply)

- [ ] Unit tests have been added or updated
- [ ] Documentation has been modified appropriately
- [ ] All ci tests pass (green)
- [ ] The [changelog](https://github.com/urbanopt/urbanopt-scenario-gem/blob/develop/CHANGELOG.md) has been updated appropriately
- [ ] This branch is up-to-date with develop

### CODE REVIEW

File: lib/urbanopt/scenario/default_reports/schema/README.md [hunk 1]
```diff
@@ -18,17 +18,16 @@ The URBANopt Scenario Gem includes functionality for defining scenarios, running
...
 |            output                        |  unit   |
 | -----------------------------------------| ------- |
-| Electricity:Facility                     | kbtu     |
-| ElectricityProduced:Facility             | kbtu     |
-| Gas:Facility                             | kbtu    |
-| DistrictCooling:Facility                 | kbtu    |
-| DistrictHeating:Facility                 | kbtu    |
+| Electricity:Facility                     | kWh     |
+| ElectricityProduced:Facility             | kWh     |
+| Gas:Facility                             | kWh     |
```
Comment (vtnate) [inline lib/urbanopt/scenario/default_reports/schema/README.md]: Are we really displaying gas in kWh units?

Comment (rawadelkontar) [reply lib/urbanopt/scenario/default_reports/schema/README.md]: Yes,  kWh is energy same as kBtu.

Comment (kflemin) [reply lib/urbanopt/scenario/default_reports/schema/README.md]: you aren't only reporting it in kWh though? kBtu is much more common...was there a particular request for gas (and electric) in kWh?

File: lib/measures/default_feature_reports/measure.rb [hunk 2]
```diff
@@ -75,13 +75,13 @@ def arguments
     reporting_frequency_chs = OpenStudio::StringVector.new
     reporting_frequency_chs << 'Detailed'
     reporting_frequency_chs << 'Timestep'
-    #reporting_frequency_chs << 'Hourly'
-    #reporting_frequency_chs << 'Daily'
+    reporting_frequency_chs << 'Hourly'
+    reporting_frequency_chs << 'Daily'
```
Comment (kflemin) [inline lib/measures/default_feature_reports/measure.rb]: remove whitespace here

**Pull request merged**